src/AppBundle/Controller/DonationController.php

```diff
@@ -21,8 +24,50 @@ class DonationController extends Controller
      */
     public function indexAction(Request $request)
     {
+        $amount = $request->query->get('montant');
+        $isMonthlyPayment = $request->get('abonnement');
```

Comment:
Usage of the `Request::get()` method is dangerous as we cannot guess the original source (Path, GET or POST) of the parameters.